strings/accent_universal.json

```
@@ -12,27 +12,22 @@
 		"humour": "humor",
 		"saviour": "savior",
 		"the earth": "creation",
-		"earth": "creation",
+		"earth": "Grimoria",
 		"penis": "cock",
-		"retarded": "a nimrod",
-		"retard": "nimrod",
-		"negro": "gnerd",
-		"nigger": "gnerd",
-		"nigga": "gnerd",
-		"niggo": "gnerd",
+		"retarded": "a dullard",
```

```suggestion
		"retarded": "a natural born idiot",
```
Period appropiate definition (14th century english)